controllers/dspipeline_controller.go

```diff
@@ -297,7 +302,14 @@ func (r *DSPAReconciler) handleReadyCondition(
 
 	err := r.Get(ctx, types.NamespacedName{Name: component, Namespace: dspa.Namespace}, deployment)
 	if err != nil {
-		return metav1.Condition{}, err
+		if apierrs.IsNotFound(err) {
+			readyCondition.Reason = config.ComponentDeploymentNotFound
+			readyCondition.Status = metav1.ConditionFalse
+			readyCondition.Message = fmt.Sprintf("Deployment for component \"%s\" is missing", component)
```

Comment:
The message sounds like something went wrong, but it seems expected behavior to see this when we're waiting for db or storage to come up right? I'm just wondering if we could word this differently to sound less ominous, like "Deployment for component is waiting for pre-requisites to come up,..."

Reply:
updated